server/sonar-server/src/main/java/org/sonar/server/rule/CachingRuleFinder.java

```diff
@@ -174,7 +174,8 @@ private static Rule toRule(RuleDefinitionDto ruleDefinition, List<RuleParamDto>
       .setRepositoryKey(ruleDefinition.getRepositoryKey())
       .setSeverity(severity != null ? RulePriority.valueOf(severity) : null)
       .setStatus(ruleDefinition.getStatus().name())
-      .setTags(ruleDefinition.getSystemTags().toArray(new String[ruleDefinition.getSystemTags().size()]))
+      .setSystemTags(ruleDefinition.getSystemTags().toArray(new String[ruleDefinition.getSystemTags().size()]))
+      .setTags(new String[0])
```

Comment:
I think that this is a change of behavior. To keep backward-compatibility what about referencing system tags in both `tags` and `systemTags` ?

Reply:
I'll keep the backward compatibilty in LTS but fix it in 7.1